src/formatDate.js

```diff
@@ -8,7 +8,42 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  let dateArr = [];
+  let resultString = '';
+  let index = 0;
+
+  dateArr = date.split(fromFormat[3]);
+
+  for (let i = 0; i < toFormat.length - 1; i++) {
+    const fromIndex = fromFormat.indexOf(toFormat[i]);
+
+    index++;
+
+    if (toFormat[i] === 'YY' && fromFormat.includes('YYYY')) {
+      const number = dateArr[index - 1];
+
+      resultString += number.slice(-2);
+    } else if (toFormat[i] === 'YYYY' && fromFormat.includes('YY')) {
```

Comment:
Here you're converting from `YYYY` to `YY` format. You're correctly slicing the last two characters from the year. However, the separator is not added after the year. This can lead to incorrect results if the year is not the last part in the new format. Consider adding the separator after the year, if it is not the last part in the new format.